content/docs/operating/integrations.md

```diff
@@ -62,6 +62,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Alertsnitch](https://gitlab.com/yakshaving.art/alertsnitch): saves alerts to a MySQL database
   * [AWS SNS](https://github.com/DataReply/alertmanager-sns-forwarder)
   * [DingTalk](https://github.com/timonwong/prometheus-webhook-dingtalk)
+  * [GELF](https://github.com/b-com-software-basis/alertmanager2gelf): Prometheus Alertmanager webhook to gelf
```

Comment:
You don't need the note here, these are all webhooks to something so it's implied that this is what this is.

Reply:
I removed the note.